analytics/views/support.py

```diff
@@ -406,3 +412,55 @@ def get_realm_admin_emails_as_string(realm: Realm) -> str:
     )
 
     return render(request, "analytics/support.html", context=context)
+
+
+def get_remote_servers_table(
+    email_to_search: Optional[str], hostname_to_search: Optional[str]
+) -> str:
+    remote_servers_query = RemoteZulipServer.objects.order_by("id")
+    if email_to_search:
+        remote_servers_query = remote_servers_query.filter(contact_email__iexact=email_to_search)
+    elif hostname_to_search:
+        remote_servers_query = remote_servers_query.filter(hostname__icontains=hostname_to_search)
+
+    remote_servers = remote_servers_query.values("id", "hostname", "contact_email", "last_updated")
+    rows = []
+    for remote_server in remote_servers:
+        remote_server_hostname_with_stats = remote_installation_stats_link(
+            remote_server["id"], remote_server["hostname"]
+        )
```

Comment:
I assume this will become instead a link to the server's `/activity/remote/` page once we have that?

Reply:
Yeah, or probably better we'll want to have both links.